[2018-09-19 17:50:21 @2329.0]	ERROR	swoole_file_put_contents(:615): open(/home/marcus/lnmp/nginx/html/easywoole_api/Temp/pid.pid) failed. Error: No such file or directory[2].
loading data from file at process EasySwoole_Cache_Process_2
loading data from file success at process EasySwoole_Cache_Process_2
loading data from file at process EasySwoole_Cache_Process_0
loading data from file success at process EasySwoole_Cache_Process_0
loading data from file at process EasySwoole_Cache_Process_1
loading data from file success at process EasySwoole_Cache_Process_1
string(22) "shake success at fd :1"
string(15) "tasklist_init:1"
string(7) "close:1"
string(22) "shake success at fd :2"
string(15) "tasklist_init:2"
string(7) "close:2"
string(22) "shake success at fd :3"
string(15) "tasklist_init:3"
string(7) "close:3"
string(22) "shake success at fd :4"
string(15) "tasklist_init:4"
string(22) "shake success at fd :5"
string(15) "tasklist_init:5"
string(7) "close:5"
string(7) "close:4"
string(22) "shake success at fd :6"
string(22) "shake success at fd :7"
string(15) "tasklist_init:6"
string(15) "tasklist_init:7"
string(7) "close:6"
string(7) "close:7"
string(22) "shake success at fd :8"
string(22) "shake success at fd :9"
string(15) "tasklist_init:9"
string(15) "tasklist_init:8"
string(7) "close:9"
string(7) "close:8"
string(23) "shake success at fd :10"
string(23) "shake success at fd :11"
string(16) "tasklist_init:11"
string(16) "tasklist_init:10"
string(8) "close:11"
string(8) "close:10"
string(23) "shake success at fd :12"
string(23) "shake success at fd :13"
string(16) "tasklist_init:12"
string(16) "tasklist_init:13"
string(12) "shake fai1 1"
string(8) "close:14"
string(12) "shake fai1 1"
string(8) "close:15"
string(12) "shake fai1 1"
string(8) "close:16"
string(12) "shake fai1 1"
string(8) "close:17"
string(12) "shake fai1 1"
string(8) "close:18"
string(12) "shake fai1 1"
string(8) "close:19"
[2018-09-20 09:41:51 #3632.2]	NOTICE	Server is shutdown now.
[2018-09-20 18:20:51 *20073.7]	ERROR	zm_deactivate_swoole (ERROR 503): Fatal error: App\WebSocket\Users and App\WebSocket\Projects define the same property ($_auth_rules) in the composition of App\WebSocket\WebSocketController. However, the definition differs and is considered incompatible. Class was composed in /home/marcus/lnmp/nginx/html/easywoole_api/App/WebSocket/WebSocketController.php on line 21.
[2018-09-20 18:20:51 $20055.0]	WARNING	swManager_check_exit_status: worker#7 abnormal exit, status=255, signal=0
[2018-09-20 18:21:16 *20155.7]	ERROR	zm_deactivate_swoole (ERROR 503): Fatal error: App\WebSocket\Users and App\WebSocket\Projects define the same property ($_auth_rules) in the composition of App\WebSocket\WebSocketController. However, the definition differs and is considered incompatible. Class was composed in /home/marcus/lnmp/nginx/html/easywoole_api/App/WebSocket/WebSocketController.php on line 21.
[2018-09-20 18:21:16 $20055.0]	WARNING	swManager_check_exit_status: worker#7 abnormal exit, status=255, signal=0
[2018-09-20 18:21:18 *20176.7]	ERROR	zm_deactivate_swoole (ERROR 503): Fatal error: App\WebSocket\Users and App\WebSocket\Projects define the same property ($_auth_rules) in the composition of App\WebSocket\WebSocketController. However, the definition differs and is considered incompatible. Class was composed in /home/marcus/lnmp/nginx/html/easywoole_api/App/WebSocket/WebSocketController.php on line 21.
[2018-09-20 18:21:18 $20055.0]	WARNING	swManager_check_exit_status: worker#7 abnormal exit, status=255, signal=0
[2018-09-20 18:21:20 *20177.7]	ERROR	zm_deactivate_swoole (ERROR 503): Fatal error: App\WebSocket\Users and App\WebSocket\Projects define the same property ($_auth_rules) in the composition of App\WebSocket\WebSocketController. However, the definition differs and is considered incompatible. Class was composed in /home/marcus/lnmp/nginx/html/easywoole_api/App/WebSocket/WebSocketController.php on line 21.
[2018-09-20 18:21:20 $20055.0]	WARNING	swManager_check_exit_status: worker#7 abnormal exit, status=255, signal=0
[2018-09-20 18:55:48 ^22623.10]	ERROR	zm_deactivate_swoole (ERROR 503): Fatal error: Uncaught SuperClosure\Exception\ClosureUnserializationException: The closure did not unserialize to a SuperClosure. in /home/marcus/lnmp/nginx/html/easywoole_api/vendor/jeremeamia/SuperClosure/src/Serializer.php:110
Stack trace:
#0 /home/marcus/lnmp/nginx/html/easywoole_api/vendor/easyswoole/easyswoole/src/Core/Component/SuperClosure.php(35): SuperClosure\Serializer->unserialize('N;')
#1 [internal function]: EasySwoole\Core\Component\SuperClosure->__wakeup()
#
[2018-09-20 18:55:48 $22618.0]	WARNING	swManager_check_exit_status: worker#10 abnormal exit, status=255, signal=0
[2018-09-20 19:49:52 *26547.7]	NOTICE	swFactoryProcess_finish (ERROR 1004): send 79 byte failed, because connection[fd=1] is closed.
[2018-09-20 19:49:52 *26547.7]	NOTICE	swFactoryProcess_finish (ERROR 1004): send 85 byte failed, because connection[fd=1] is closed.
[2018-09-20 21:41:59 #26528.2]	NOTICE	Server is shutdown now.
[2018-09-20 23:03:31 *6606.7]	WARNING	swWorker_discard_data (ERROR 1007): [1]received the wrong data[99 bytes] from socket#4
[2018-09-20 23:03:53 *6606.7]	WARNING	swWorker_discard_data (ERROR 1007): [1]received the wrong data[99 bytes] from socket#5
[2018-09-20 23:18:50 *7767.7]	NOTICE	swFactoryProcess_finish (ERROR 1004): send 216 byte failed, because connection[fd=1] is closed.
[2018-09-20 23:23:29 *8227.7]	NOTICE	swFactoryProcess_finish (ERROR 1004): send 200 byte failed, because connection[fd=1] is closed.
[2018-09-20 23:56:10 *10471.7]	NOTICE	swFactoryProcess_finish (ERROR 1004): send 228 byte failed, because connection[fd=1] is closed.
[2018-09-20 23:56:10 *10471.7]	NOTICE	swFactoryProcess_finish (ERROR 1004): send 473 byte failed, because connection[fd=1] is closed.
loading data from file at process EasySwoole_Cache_Process_2
loading data from file success at process EasySwoole_Cache_Process_2
loading data from file at process EasySwoole_Cache_Process_1
loading data from file success at process EasySwoole_Cache_Process_1
loading data from file at process EasySwoole_Cache_Process_0
loading data from file success at process EasySwoole_Cache_Process_0
NULL
string(7) "close:1"
[2018-09-21 15:08:41 #15509.2]	NOTICE	Server is shutdown now.
[2018-09-21 17:41:36 *4355.7]	NOTICE	swFactoryProcess_finish (ERROR 1004): send 295 byte failed, because connection[fd=2] is closed.
[2018-09-21 17:41:36 *4355.7]	NOTICE	swFactoryProcess_finish (ERROR 1004): send 712 byte failed, because connection[fd=2] is closed.
[2018-09-21 20:37:35 *6683.7]	WARNING	swSignalfd_onSignal: signal[1] callback is null.
[2018-09-21 20:37:35 *6682.6]	WARNING	swSignalfd_onSignal: signal[1] callback is null.
[2018-09-21 20:37:35 *6680.4]	WARNING	swSignalfd_onSignal: signal[1] callback is null.
[2018-09-21 20:37:35 *6677.1]	WARNING	swSignalfd_onSignal: signal[1] callback is null.
[2018-09-21 20:37:35 $6665.0]	WARNING	swSignal_callback: signal[1] callback is null.
[2018-09-21 20:37:35 #6664.2]	WARNING	swSignalfd_onSignal: signal[1] callback is null.
[2018-09-21 20:37:35 $6665.0]	WARNING	swManager_check_exit_status: worker#18 abnormal exit, status=0, signal=1
[2018-09-21 20:37:35 $6665.0]	WARNING	swManager_check_exit_status: worker#17 abnormal exit, status=0, signal=1
[2018-09-21 20:37:35 *6681.5]	WARNING	swSignalfd_onSignal: signal[1] callback is null.
[2018-09-21 20:37:35 *6679.3]	WARNING	swSignalfd_onSignal: signal[1] callback is null.
[2018-09-21 20:37:35 *6678.2]	WARNING	swSignalfd_onSignal: signal[1] callback is null.
[2018-09-21 20:37:35 $6665.0]	WARNING	swManager_check_exit_status: worker#16 abnormal exit, status=0, signal=1
[2018-09-21 20:37:35 *6676.0]	WARNING	swSignalfd_onSignal: signal[1] callback is null.
[2018-09-25 09:10:00 #6664.2]	NOTICE	Server is shutdown now.
